cypress/e2e/tests/pages/explorer/workloads/pods.spec.ts

```diff
@@ -7,41 +7,50 @@ import { generatePodsDataSmall } from '@/cypress/e2e/blueprints/explorer/workloa
 
 describe('Pods', { testIsolation: 'off', tags: ['@explorer', '@adminUser'] }, () => {
   const workloadsPodPage = new WorkloadsPodsListPagePo('local');
-  const nsName = `namespace${ +new Date() }`;
-  const nsName2 = `namespace${ +new Date() }abc`;
   const uniquePod = 'aaa-e2e-test-pod-name';
   const podNamesList = [];
 
   before(() => {
     cy.login();
   });
 
+  beforeEach(() => {
+    cy.createE2EResourceName('namespace1').as('namespaceName1');
+    cy.createE2EResourceName('namespace2').as('namespaceName2');
+  });
+
   describe('List', { tags: ['@vai'] }, () => {
     before('set up', () => {
-      cy.updateResourceListViewPref('local', 'none', '{\"local\":[]}');
+      cy.updateNamespaceFilter('local', 'none', '{\"local\":[]}');
 
-      // create namespaces
-      cy.createNamespace(nsName);
-      cy.createNamespace(nsName2);
+      cy.createE2EResourceName('namespace1').then((nsName1) => {
```

Comment:
this and below have already been created in the beforeEach

Reply:
correct, but the before() hooks run prior to the beforeEach() so when we only create it in the beforeEach() an error is thrown (e.g. `namespace1` is not found). To resolve this, I'll declare the variable prior to the before() and get rid of the beforeEach() hook.